node/src/options.rs

```diff
@@ -103,23 +109,23 @@ pub struct RunOptions {
 
     /// The p2p timeout value in seconds.
     #[clap(long)]
-    pub p2p_outbound_connection_timeout: Option<NonZeroU64>,
+    pub p2p_outbound_connection_timeout: Option<RpcTime>,
```

Comment:
I don't think using RpcTime for these variables is right. Time is an absolute value, and timeout is a difference in time. These are, by coincidence, the same type. But they shouldn't be the same type while being used.

In fact, the NonZeroU64 is there for a reason. Now this is broken.

Reply:
I guess the name `RpcTime` was not appropriate. I changed it to `RpcDuration`. It is used where the `Duration` type is expected (but where `Duration` itself cannot be used). In any case, it should be better than the currently used `u32` and `u64`.